rq/group.py

```diff
@@ -17,34 +17,34 @@ class Group:
     REDIS_GROUP_NAME_PREFIX = 'rq:group:'
     REDIS_GROUP_KEY = 'rq:groups'
 
-    def __init__(self, connection: Redis, id: str = None):
-        self.id = id if id else str(uuid4())
+    def __init__(self, connection: Redis, name: str = None):
+        self.name = name if name else str(uuid4())
```

Comment:
Nitpick, let's make this `uuid4().hex` so it's a bit shorter.

Reply:
Done